### DISCUSSION TITLE
I have a textbox width = 230 ,if this text has chinese or japanese or A continuous letter how to deal with wrapline？

### DISCUSSION THREAD

**Discussion by m3360202** [fabricjs/fabric.js#9093] [General]
if english word mix with chinese or japanese or A continuous letter,it seems that wrapline also not work well

https://github.com/fabricjs/fabric.js/assets/36294167/42db8af7-f017-43c7-bdbb-1eadc1df7a46

that's how to Recurrence issues

**Comment by m3360202** [comment]
this is base on fabric js 6.0.0 beta5,i've already set splitByGrapheme=false cause i need Ensure that English words are not broken when wrapping

**Reply by ShaMan123** [reply]
Try beta10 and override `getGraphemeDataForRender`
Split up how you want and it should work

**Reply by m3360202** [reply]
it's not work，if there has  a text 'this is the ice cream that aaaaaaaaaaaaaaaaaaaaa aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa' in a width 230 textbox,the wraptext go wrong

**Reply by ShaMan123** [reply]
Your reply is meaningless
You haven't shown your attemp
I do NOT wish to waste my time

**Comment by m3360202** [comment]
What I mean is that in other languages such as Chinese or Japanese, there are no spaces between each word，it will be considered as a long character word， Therefore, if I have a section of text that contains both English and other languages, and set splitByGraphme to false, I hope that English words can be folded normally. At the same time, I hope that text in other languages can be forced to wrap when it reaches the end of a paragraph in the textbox and the entire length of the text is about to exceed the width of the textbox
